recipe/meta.yaml

```diff
@@ -55,7 +55,6 @@ requirements:
     - {{ mpi }}  # [mpi != 'nompi']
     - mpich-mpicc  # [mpi == 'mpich']
     - mpi4py  # [mpi != 'nompi']
-    - libssh2 1.8.*
```

Comment:
As discussed here: https://github.com/conda-forge/netcdf4-feedstock/pull/101#discussion_r425965497
This was just a test and shouldn't have been merged.